scripts/test_nearlib.sh

```diff
@@ -24,11 +27,12 @@ yarn build
 yarn test
 yarn doc
 
-# Run near-shell tests
-# cd ../near-shell
-# yarn && yarn test
-
 # Run create-near-app tests
-# cd ../create-near-app
-# yarn && yarn test
+cd ../create-near-app
+yarn 
+yarn test
 
+# Run near-shell tests
```

Comment:
@ailisp do you plan to enable these in follow up PR?

Reply:
This still fails of missing masterAccount param, will fix later